Assets/Assets/PlanetTerrain/TEST.asset

```diff
@@ -22,7 +22,7 @@ RenderTexture:
   m_ColorFormat: 49
   m_MipMap: 0
   m_GenerateMips: 1
-  m_SRGB: 1
```

Comment:
Is this asset still used? Maybe you could remove it if you are not using it:)

Reply:
Otherwise I think it looks good. So I can merge it after you answer this little thing.

Reply:
No, they are indeed old, I have removed them now and pushed up the branch

Reply:
Nice! Good job!